ospf6d/ospf6_neighbor.c

```diff
@@ -1004,6 +1004,8 @@ static void ospf6_neighbor_show_detail(struct vty *vty,
 				    on->ospf6_if->interface->ifindex);
 		json_object_int_add(json_neighbor, "neighborInterfaceIndex",
 				    on->ifindex);
+		json_object_string_addf(json_neighbor, "localIfaceAddress",
```

Comment:
Wouldn't be better to name it `linkLocalAddress`?

Reply:
Well it does beg the question, what the next json_object_string_add below is and how it is different than the addition of this particular field.

Reply:
ah the linklocal_addr as set on 988 is the neighbors LL address.  Louis is attempting to show this router's interface LL address.

Reply:
Then `localLinkLocalAddress`? :smile:

Reply:
Seems ok I guess